File: App/src/app/model/Inventorydto.ts
//galing ui
export interface CreateInventoryRequest{

  productName: string;
  availableQty: number;
  reorderPoint: number;
}
export interface UpdateInventoryRequest{
  id: number;
  productName: string;
  availableQty: number;
  reorderPoint: number;
}
export interface DeleteInventoryRequest{
  id: number;
}
//galing server or Response from server
export interface Inventorydto{
  id: number;
  productName: string;
  availableQty: number;
  reorderPoint: number;
} 

File: AppWebApi/Application/Dto/Inventory/Response/Inventorydto.cs
namespace Application.Dto.Inventory.Response
{
    public class Inventorydto
    {
        //galing server
        public int Id { get; set; }
        public string? ProductName { get; set; }
        public int AvailableQty { get; set; }
        public int ReorderPoint { get; set; }
    }
}
